pkg/test/ginkgo/synthentic_tests.go

```diff
@@ -73,11 +83,52 @@ func createSyntheticTestsFromMonitor(events monitorapi.Intervals, monitorDuratio
 			// write a passing test to trigger detection of this issue as a flake, indicating we have no idea whether
 			// these are actual failures or not
 			&JUnitTestCase{
-				Name:     "[sig-arch] Monitor cluster while tests execute",
+				Name:     monitorTestName,
 				Duration: monitorDuration.Seconds(),
 			},
 		)
+	} else {
+		// even if no error events, add a passed test including the output so we can scan with search.ci:
+		syntheticTestResults = append(
+			syntheticTestResults,
+			&JUnitTestCase{
+				Name:     monitorTestName,
+				Duration: monitorDuration.Seconds(),
+				SystemOut: buf.String(),
+			},
+		)
+	}
+
+	errImagePullTestName :="[sig-arch] No ErrImagePull read connection timeout intervals"
+	if errImagePullTimeouts > 0 {
+		syntheticTestResults = append(
+			syntheticTestResults,
+			&JUnitTestCase{
+				Name:     errImagePullTestName,
+				SystemOut: buf.String(),
+				Duration:  monitorDuration.Seconds(),
+				FailureOutput: &FailureOutput{
+					// Not including the entire buffer again as it's very large and is added to junit xml above.
+					Output: fmt.Sprintf("%d ErrImagePull read: connection timed out events were detected during this test run", errImagePullTimeouts),
```

Comment:
very reasonable test.  Try to factor it like https://github.com/openshift/origin/blob/master/pkg/synthetictests/event_junits.go#L18

and capture:
1. errimagepull in openshift-* namespaces for any reason
2. errimagepull in openshift-* namespaces for read connection timed out
3. errimagepull in non-openshift-* namespaces for any reason
4. errimagepull in non-openshift-* namespaces for read connection timed out

as four different tests.  you may want to start all of them as flaking.